packages/route-viewer-overlay/src/index.tsx

```diff
@@ -136,11 +137,18 @@ const RouteViewerOverlay = (props: Props): JSX.Element => {
     });
 
     if (bounds && current) {
-      util.fitMapBounds(current, bounds);
+      // Use a timeout to wait for other map rendering to finish
+      // before fitting to the route or pattern shape.
```

Comment:
I think this comment could be adjusted to explain why the function is being called twice. (reducing perceived latency)